cumulusci/tasks/salesforce/activate_flow.py

```diff
@@ -20,7 +20,8 @@ def _init_options(self, kwargs):
         self.options["developer_names"] = process_list_arg(
             self.options.get("developer_names")
         )
-        self.api_version = "43.0"
+        # TODO: IS THIS RIGHT????
+        self._api_version = "43.0"
```

Comment:
Looks like this was just the API version that was current when the task was written. Needs a smoke test.